### PR TITLE
Fix AutocompleteInput choices are displayed again after selection

### LINKED ISSUE #2428: AutocompleteInput pop-up after choice is selected
**What you were expecting:**
After choice is selected, AutocompleteInput filled the textInput with selected choice

**What happened instead:**
After choice is selected, AutocompleteInput filled the textInput with selected choice then pop-up all the choices again

**Steps to reproduce:**
1. Goto demo site, to edit an order (e.g. https://marmelab.com/react-admin-demo/#/commands/280)
2. Enter a customer name in the Customer field
3. Select the choice with your mouse
4. All choices were pop-up again

Please see the attached GIF for the issue
autocomplete

### PR DESCRIPTION
**Pull request by djhi**
Fixes #2428

### CODE REVIEW

File: cypress/integration/edit.js [hunk 1]
```diff
@@ -78,6 +78,29 @@ describe('Edit Page', () => {
         );
     });
 
+    it('should allow to select an item from the AutocompleteInput without showing the choices again after', () => {
+        EditCommentPage.navigate();
+        cy.get(EditPostPage.elements.input('post_id')).clear().type('Sed quo');
```
Comment (fzaninotto) [inline cypress/integration/edit.js]: I don't understand: you use the EditPostPage here but you loaded the EditCommentPage just above...

Comment (djhi) [reply cypress/integration/edit.js]: It actually shows that those page objects are confusing. In this case, both were instantiated at the top of the test file. They both retrieve the correct item from the page with this `input` function. 

Anyway, *fixed* it

**Comment by wmwart** [comment]
Not help. Still continues to open after upgrade. In the Demo, too.(
https://marmelab.com/react-admin-demo/#/commands/477

**Comment by Kmaschta** [comment]
@wmwart Mind you open/reopen an issue with a reproducible example please?

**Comment by wmwart** [comment]
Does everything work for you?

**Comment by kctrlv** [comment]
@wmwart I'm experiencing the same issue, not yet fixed as of 2.8.2

**Comment by Chris-Sytadelle** [comment]
I am experiencing the same issue with 2.8.2.

**Pull request merged**